generate/generate.go

```diff
@@ -128,15 +126,66 @@ func Do(root string) error {
 	return nil
 }
 
-func generateStackConfig(root string, configdir string, evalctx *eval.Context) ([]byte, error) {
+func generateStackLocals(
+	rootdir string,
+	stackpath string,
+	metadata terramate.StackMetadata,
+	globals *terramate.Globals,
+) error {
+	stackLocals, err := terramate.LoadStackExportedLocals(rootdir, metadata, globals)
+	if err != nil {
+		return err
+	}
+
+	localsAttrs := stackLocals.Attributes()
+	if len(localsAttrs) == 0 {
+		return nil
+	}
+
+	sortedAttrs := make([]string, 0, len(localsAttrs))
+	for name := range localsAttrs {
+		sortedAttrs = append(sortedAttrs, name)
+	}
+	// Avoid generating code with random attr order (map iteration is random)
+	sort.Strings(sortedAttrs)
```

Comment:
maybe in the future, it would make sense to keep the user declared order in the `export_as_locals` map.